tests/shell/tests/01-run.py

```diff
@@ -60,6 +59,8 @@
     ('end_test', '[TEST_END]'),
 )
 
+CMDS_REGEX = { 'ps' }
```

Comment:
Flake8 wants it like this:
```suggestion
CMDS_REGEX = {'ps'}
```

Reply:
Feel free to amend this directly

Reply:
I foresee a conflict with #13197, as a headsup.

Reply:
seems like git manages to merge this just fine. :smile:

Reply:
fixed